README.md

```diff
@@ -2,55 +2,55 @@
 
 # Project Name
 
-Description the project.
+<<<<<<< HEAD
+- This is an application that displays a list of Covid Data generated for the african continent from an API.
 
+## Video Presentation
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+[Presentation](https://www.loom.com/share/bb3c405e43ce48228658dd2b115c6e24)
```

Comment:
* Welldone 👏 adding a link to the recorded video presentation, but your video should contain a highlight about a piece of the code the student finds interesting or is proud of making.  please 🙏 add this to your video presentation as it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.)